Dockerfile

```diff
@@ -19,11 +19,11 @@ COPY ./db/wait-for /app/wait-for
 
 ADD ./src/main/resources/template /app/template
 
-RUN chmod +x /app/wait-for
-RUN chmod +x /app/verify/verify
+RUN chmod +x /app/wait-for /app/verify/verify  \
+    && apt-get update  \
+    && apt-get install -y --no-install-recommends netcat  \
+    && rm -rf /var/lib/apt/lists/*  \
```

Comment:
Clean up the apt cache by removing `/var/lib/apt/lists` it reduces the image size, since the apt cache is not stored in a layer.